nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/StorageConfiguration.java

```diff
@@ -19,6 +19,7 @@
 package org.nuxeo.ecm.core.test;
 
 import static org.junit.Assert.assertNotNull;
+import static org.nuxeo.runtime.stream.RuntimeStreamFeature.defaultProperty;
```

Comment:
`StorageConfiguration` and `RuntimeStreamFeature` are totally unrelated so it's weird to static import a helper from there. Just keep the method here, it's a small duplication but keeps things consistent. We'll refactor at some point when all these "test configuration services" get rethought into a cleaner global test config framework. 

Same in `WorkManagerFeature`.